plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -229,6 +229,26 @@ func (e *Elasticsearch) createHttpClient() (*http.Client, error) {
 	return client, nil
 }
 
+func (e *Elasticsearch) generateIndividualStatsUrlPart() string {
```

Comment:
Rename `nodeStatsUrlPart(`, or maybe just integrate into the parent function since it is not called from anywhere else.